### ISSUE TITLE
Adding rooms

### ISSUE THREAD

**Issue by ChrisIck** [iranroman/SELD-data-generator#6]
Right now, the database is hard-coded for use with the TAU-SRIR database. @adrianSRoman has already added an additional room in #2 , but we should look into how we can add rooms in a scalable way.

**Comment by ChrisIck** [comment]
Pending on #1 for conversion to `.yml` files for database config, so will leave this on hold for now.

**Comment by ChrisIck** [comment]
This _should_ be covered by #20, and will be further addressed in #17

**Comment by iranroman** [comment]
We should NOT add MARCo. From what I understand, in MARCo the Eigenmike was tilted toward the stage and correcting for this factor may be tricky. 

@karinnacvijanovich has opened a PR #69 that adds three more rooms.

**Comment by iranroman** [comment]
This dataset is also great and we should add it:

https://zenodo.org/records/6257551#.YhfMwC-l30o

Volunteers? @abreuwallace @sripathisridhar @sakshamsingh1 @aidanasingh
